### PR TITLE
Remove ghprb vars from bml pipeline

### PR DESCRIPTION
**Pull request by Sunnatillo**
Removes ghprb related vars from bare metal lab pipeline

### CODE REVIEW

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Comment by Rozzii** [comment]
LGTM

File: jenkins/jobs/bml_integration_tests.pipeline [hunk 1]
```diff
@@ -32,30 +33,21 @@ pipeline {
...
-    CAPM3_VERSION = "${CAPM3_VERSION}"
     CAPM3RELEASEBRANCH = "${capm3_release_branch}"
     BMORELEASEBRANCH = "${bmo_release_branch}"
     TARGET_NODE_MEMORY = "${TARGET_NODE_MEMORY}"
-    NUM_NODES="${NUM_NODES}"
-    GINKGO_FOCUS="${GINKGO_FOCUS}"
-    GINKGO_SKIP="${GINKGO_SKIP}"
-    EPHEMERAL_TEST="${EPHEMERAL_TEST}"
-    TESTS_FOR="${TESTS_FOR}"
-    BARE_METAL_LAB=true
```
Comment (tuminoid) [inline jenkins/jobs/bml_integration_tests.pipeline:54]: Are all of these L54-L58 also obsolete and supposed to be removed?

Comment (Sunnatillo) [reply jenkins/jobs/bml_integration_tests.pipeline:54]: yes, first three is related to e2e tests. And the BARE_METAL_LAB not needed bcs we hve now separate pipeline for each type of tests.

**Review by tuminoid — COMMENTED** [review]
/approve

**Pull request merged**